Formula/xonsh.rb

```diff
@@ -3,8 +3,8 @@ class Xonsh < Formula
 
   desc "Python-powered, cross-platform, Unix-gazing shell language and command prompt"
   homepage "https://xon.sh/"
-  url "https://files.pythonhosted.org/packages/7f/3d/a29fe1f9dc61502a0cdee70e4bd4f97164106975209b6feaa2e064d241fc/xonsh-0.9.25.tar.gz"
-  sha256 "20af3480d6ac1b8a48f03288796b16a35f498f047e3472c300e4ef5a26f03d60"
+  url "https://github.com/xonsh/xonsh/archive/0.9.26.tar.gz"
```

Comment:
Please keep the pypi URL

Reply:
Oh! My bad I had not noticed the formula had been switched to use the PyPI URL! Updated, and the hash as well.

@dtrodrigues Thanks for the updates command. Time to retire the one I had in my history 😉